conf/environments.js

```diff
@@ -60,6 +60,12 @@ export default new Map(Object.entries({
     builtin: {
         globals: globals.es5
     },
+    es5: {
+        globals: globals.es5,
+        parserOptions: {
+            ecmaVersion: 5
+        }
+    },
```

Comment:
```suggestion
```

I don't think we should add `es5` environment, because it's effectively a no-op that can only cause confusion, and the concept of environments will be removed in the new config system anyway.

Current version of ESLint always adds es5 globals and has `ecmaVersion: 5` as the default, so setting `"env": { "es5": true }` would do nothing. Also, user might expect that `"env": { "es5": false }` disables diff between es5 and es3 globals, but it would do nothing as well.

Reply:
good point! 👍 

To lint es3 code(I guess it's @ljharb's use case), it should be working to use `languageOptions.ecmaVersion: 3` in the new config?

Reply:
atm it was not easy to lint es3, as no way to config es3 globals.

Reply:
Alrighty, I’ll remove es5.